packages/bff/src/model/domain/errors.ts

```diff
@@ -1,13 +1,20 @@
 import {
   ApiError,
   PurposeId,
+  TenantId,
+  EServiceId,
   makeApiProblemBuilder,
+  DescriptorId,
+  AttributeId,
 } from "pagopa-interop-models";
 
 export const errorCodes = {
   purposeNotFound: "0001",
   userNotFound: "0002",
   selfcareEntityNotFilled: "0003",
+  descriptorNotFound: "0006",
+  attributeNotExists: "0008",
+  invalidEserviceRequester: "0009",
```

Comment:
Doubt: in the BFF do we have to use the same codes used in Scala, being exposed directly to the FE, or can we do like in the processes and redefine our codes from scratch just incrementing whenever we add a new error?
cc @galales

Reply:
As discussed offline with @Carminepo2 and @beetlecrunch, this code could be different from the Scala codebase

Reply:
Perfect, so let's do the increment by one since we don't need to adhere to Scala codes:

```suggestion
  descriptorNotFound: "0004",
  attributeNotExists: "0005",
  invalidEserviceRequester: "0006",
```